tests/integration/list/resources/test_resources_command.py

```diff
@@ -93,4 +74,80 @@ def test_invalid_template_file(self):
             stack_name=None, region=region, output="json", template_file=template_path
         )
         command_result = run_command(cmdlist, cwd=self.working_dir)
-        self.assertIn("Template provided was invalid SAM Template.", command_result.stdout.decode())
+        self.assertIn(
+            "Error: [InvalidTemplateException(\"'Resources' section is required\")] 'Resources' section is required",
+            command_result.stderr.decode(),
+        )
+
+    def test_success_with_stack_name(self):
+        template_path = self.list_test_data_path.joinpath("test_stack_creation_template.yaml")
+        stack_name = method_to_stack_name(self.id())
+        config_file_name = stack_name + ".toml"
+        region = boto3.Session().region_name
+        deploy_command_list = self.get_deploy_command_list(
+            template_file=template_path,
+            guided=True,
+            config_file=config_file_name,
+            region=region,
+            confirm_changeset=True,
+            disable_rollback=True,
+        )
+        deploy_process_execute = run_command_with_input(
+            deploy_command_list, "{}\n{}\nY\nY\nY\nY\nY\n\n\nY\n".format(stack_name, region).encode()
+        )
+        cmdlist = self.get_resources_command_list(
+            stack_name=stack_name, region=region, output="json", template_file=template_path
+        )
+        command_result = run_command(cmdlist, cwd=self.working_dir)
+        self.assertTrue(
```

Comment:
nit: Not a big deal, but I wonder if we could have an array of expected regex and loop through it to do the assertions instead of doing them one-by-one. Could make it a little cleaner and more compact.

Reply:
Changed to regex list